### ISSUE TITLE
Remove redundant "purescript-" prefix

### ISSUE THREAD

**Issue by ad-si** [purescript/pursuit#433]
By now there are no more packages without the "purescript-" prefix on Pursuit. So maybe we should just not show it anymore? It adds a lot of visual noise and the listing would be much nicer without it.

What is actually the future of the "purescript-" prefix? If I remember correctly this was a way to avoid name collisions with JavaScript packages on Bower, but with a switch to purescript/registry this won't be necessary anymore, right?

**Comment by hdgarrood** [comment]
Yes, it won’t be necessary with the new registry. I’d like to revisit this once the new registry is further along.
